server/src/concepts/user/controllers/index.js

```diff
@@ -13,3 +14,12 @@ export const createUser = async (req, res) => {
   }
   return res.status(200).send(`User with ${req.body.email} was created`);
 };
+
+export const deleteUser = async (req, res) => {
+  try {
+    await deleteUserWithId(req.params.id);
```

Comment:
```suggestion
    const { deletedCount } = await deleteUserWithId(req.params.id);
    if(!deletedCount) return res.status(404).send("There is no user")
```

here is good place for that.
Why deletedCount? function User.deleteOne() (from commands) return object: { deletedCount: number }, so we are destructuring this object and then check if is more than 0. **IMPORTANT: Please return value from deleteUser in commands and other commands as well.**